packages/office-addin-telemetry/README.md

```diff
@@ -16,7 +16,7 @@ The package collects:
 * [On](#On)
 
 ### List
-Display the current telemetry settings.
+Display the current usage-data settings.
```

Comment:
"usage data"  Please go throughout and fix places where "usage-data" should be two words.